lib/takeaway_kitchen.rb

```diff
@@ -0,0 +1,19 @@
+class TakeawayKitchen
+
```

Comment:
I like how the menu is in a separate class. It makes everything clean and tidy when looking at the ordersystem file.